pkg/util/validator.go

```diff
@@ -13,8 +13,8 @@ import (
 )
 
 func ValidateSubnet(subnet kubeovnv1.Subnet) error {
-	if subnet.Spec.Gateway != "" && !CIDRContainIP(subnet.Spec.CIDRBlock, subnet.Spec.Gateway) {
-		return fmt.Errorf(" gateway %s is not in cidr %s", subnet.Spec.Gateway, subnet.Spec.CIDRBlock)
+	if subnet.Spec.Gateway != "" && (!CIDRContainIP(subnet.Spec.CIDRBlock, subnet.Spec.Gateway) || isNetworkOrBroadcast(subnet.Spec.CIDRBlock, subnet.Spec.Gateway)) {
```

Comment:
Can we use `ValidatePodCidr` here?

Reply:
sure，i think It is more appropriate to change the function name ValidatePodCidr to ValidateNetworkBroadcast